components/report/ReportModal.vue

```diff
@@ -69,6 +86,22 @@ async function submitReport() {
   resetModal()
 }
 
+async function submitAnonymousReport() {
+  const body = JSON.stringify({
+    account_id: account.id,
+    status_ids: selectedStatusIds.value,
+    comment: additionalComments.value,
+    forward: forwardReport.value,
+    category: reportReason.value === 'spam' ? 'spam' : reportReason.value === 'violation' ? 'violation' : 'other',
```

Comment:
Ternary nesting is a little hard to mentally parse, but not a blocker.

Reply:
Agreed! Since I copied this from the existing function, I think I may just let sleeping dogs lie.